answerking_app/tests/test_categories.py

```diff
@@ -273,9 +282,9 @@ def test_put_valid_without_items_returns_ok(self):
         # Arrange
         old_category = client.get(f"/api/categories/{self.test_cat_1.id}").json()
```

Comment:
does this not need a type?

Reply:
Haven't typed those as they are quite generic and tested with the assertion, but yeah, I can add them

Reply:
I actually checked the type now and I don't think it's worth the effort, they are quite messy objects


for instance:
```
def put(self,
        path: Any,
        data: str = "",
        content_type: str = "application/octet-stream",
        follow: bool = False,
        secure: bool = False,
        **extra: Any) -> WSGIRequest | {redirect_chain, status_code, url}
```

Reply:
ah ok. I'm happy for these to be left  untyped